starboard/shared/starboard/media/iamf_util_test.cc

```diff
@@ -168,6 +167,31 @@ TEST(IamfUtilTest, Invalid) {
   util = IamfMimeUtil(codec_param);
   EXPECT_FALSE(util.is_valid());
 
+  // Negative primary profile value
+  codec_param = "iamf.-12.000.Opus.";
```

Comment:
This particular test case has a trailing '.', so it will always fail whether the implementation can parse negative profile correctly or not.